docs/reference/filebeat/filebeat-input-entity-analytics.md

```diff
@@ -272,6 +337,9 @@ The number of records to request from the Active Directory server for each page,
 
 The client’s password, used for authentication. Field is required.
 
+#### `dataset` [_ad_dataset]
+
+The datasets to collect from Active Directory. This can be one of "all", "users" or "devices", or may be left empty for the default behavior which is to collect all entities. When the `dataset` is set to "devices", some user entity data is collected in order to populate the registered users and registered owner fields for each device.
```

Comment:
Consider.

```suggestion
The datasets to collect from Active Directory. This can be one of "all", "users" or "devices", or may be left empty for the default behavior, which is to collect all entities. When the `dataset` is set to "devices", some user entity data is collected in order to populate the registered users and registered owner fields for each device.
```